src/main/java/org/broadinstitute/hellbender/engine/filters/MappingQualityReadFilter.java

```diff
@@ -10,8 +10,10 @@
 /**
  * Keep reads with mapping qualities within a specified range.
  *
- * Note: this filter does not handle specially the unavailable mapping quality ({@link org.broadinstitute.hellbender.utils.QualityUtils#MAPPING_QUALITY_UNAVAILABLE}).
- * Use {@link org.broadinstitute.hellbender.engine.filters.ReadFilterLibrary.MappingQualityAvailableReadFilter} to explicitly filter out reads with unavailable quality.
+ * <p>Note: this filter is not designed to handle the unavailable mapping quality (255).
+ * Use MappingQualityAvailableReadFilter to explicitly filter out reads with unavailable quality.
+ * @see org.broadinstitute.hellbender.utils.QualityUtils#MAPPING_QUALITY_UNAVAILABLE
+ * @see org.broadinstitute.hellbender.engine.filters.ReadFilterLibrary.MappingQualityAvailableReadFilter
  */
 @DocumentedFeature(groupName= HelpConstants.DOC_CAT_READFILTERS, groupSummary=HelpConstants.DOC_CAT_READFILTERS_SUMMARY)
```

Comment:
I added them in the `extraDocs` as a temporary solution, because with the current doclet implementation it is difficult to do something like that. I have some ideas regarding this issue:

1. Populate the information in the `@see` tag into the `extraDocs` (if they are documented). This will allow having the javadoc information for developers and the help for the user. I will open a ticket in Barclay for this one, because I think that it makes sense for everyone.
2. Another idea is to use the javadoc `{@link Foo}` tag and parse it inside the doclet (either in Barclay or in the custom one) to find the class. If it is documented, then the URL could be obtained and added (with html or markdown syntax); if it is not documented, the parsing could remove the tag marker and the qualifier to make it more readable for the user.